lr-cli/cortx_setup/commands/node/prepare/network.py

```diff
@@ -70,14 +70,12 @@ class NodePrepareNetwork(Command):
             'type': str,
             'nargs': '+',
             'optional': True,
-            'default': "",
             'help': 'DNS server'
```

Comment:
Please update help to:
'List of DNS servers for the provided network, given space-separated'

Reply:
Updated help message